playground/infrastructure/cd_helper.py

```diff
@@ -22,16 +24,30 @@ class CDHelper:
     It is used to save beam examples/katas/tests and their output on the Google Cloud.
     """
 
-    def __init__(self):
-        self.ci_helper = CIHelper()
+    def get_examples_and_store(self):
```

Comment:
This is not the best naming, it says that you should split this function into 2. I see that you already have `find_examples` as a separate function, which is good. Let's rename this function to `store_examples`, which will get examples list as an argument and then will run code and save to cloud.

Reply:
I agree with with Artur, can be store_examples if examples as argument

Reply:
Changed.